spring/pom.xml

```diff
@@ -18,7 +18,7 @@
 
 	<properties>
 		<java.version>11</java.version>
-		<zeebe.version>1.1.3</zeebe.version>
+		<zeebe.version>1.2.0</zeebe.version>
```

Comment:
:x: The Spring Zeebe Starter didn't [released](https://github.com/camunda-community-hub/spring-zeebe/releases) a version for `1.2.0` yet. 

The latest version is `1.1.0`. I guess that we need to update this version later.

Reply:
Thanks for the info.

Reply:
I fixed the spring version to 1.1.0